openapi-java-wes-client/src/main/resources/workflow-execution_service-1.0.0-oas3-openapi.yaml

```diff
@@ -199,7 +199,7 @@ paths:
               properties:
                 workflow_params:
                   type: string
-                  format: binary
+                  format: application/json
```

Comment:
Is this file a copy of the spec, and now it's differing?

If so, how will this ultimately be resolved? Also, it might be worth a comment in the commit message saying why you are making this change.

Reply:
No, this change brings the new version in line with the specification

Reply:
I feel like this is worth a comment in the commit message (maybe it's already there) and/or in OPENAPI_CREATION.md. Why wasn't it in line with the spec before? Was there an error in the conversion process? If there was an error in the conversion process, it should be documented so that when there is an update, this bug doesn't come back.

If there's actually a new version of the specification, shouldn't this have a different name?

Reply:
There was no error in the conversion process. I mistakenly changed it after the conversion to 3.0. Then I realized the format should stay as the specification described it so I fixed it.  The original schema has been updated to OpenApi 3.0.